contentcuration/contentcuration/serializers.py

```diff
@@ -76,6 +76,7 @@ def update(self, instance, validated_data):
         user = self.context['request'].user
         with transaction.atomic():
             for item in validated_data:
+                # import ipdb; ipdb.set_trace()
```

Comment:
Delete!

Reply:
Rule #1 of pdb: we don't talk about pdb (even though we all use it)

Reply:
This is the breakpoint we were using the other day.  No need to use it, I just kept it there for context.